lib/Doctrine/ORM/Mapping/Driver/AnnotationDriver.php

```diff
@@ -10,6 +10,7 @@
 use Doctrine\ORM\Events;
 use Doctrine\ORM\Mapping;
 use Doctrine\ORM\Mapping\Builder\EntityListenerBuilder;
+use Doctrine\ORM\Mapping\ClassMetadata as Metadata;
```

Comment:
I find this alias confusing. Let's use `OrmClassMetadata` or better: Alias `ClassMetadata` from Persistence as `PersistenceClassMetadata` and use `ClassMetadata` for our own implementation.